stable/kommander/requirements.lock

```diff
@@ -4,18 +4,18 @@ dependencies:
   version: 0.1.15
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.6.9
+  version: v0.6.9
 - name: kommander-licensing
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.6.9
+  version: v0.6.9
```

Comment:
```suggestion
  version: 0.6.9
```

don't think this change is needed

Reply:
its not, but `hlem dep up` sometimes does it… we dont really know why it happens, but its not part of the problem.

Reply:
yeah, not part of the problem.